src/app/frontend/common/services/global/theme.ts

```diff
@@ -14,19 +14,25 @@
 
 import {EventEmitter, Injectable} from '@angular/core';
 import {ThemeSwitchCallback} from '@api/frontendapi';
+import {Theme} from '@api/backendapi';
 
 @Injectable()
 export class ThemeService {
-  private isLightThemeEnabled_ = true;
-  private readonly onThemeSwitchEvent_ = new EventEmitter<boolean>();
+  private _availableThemes = [Theme.Light, Theme.Dark];
```

Comment:
In the future this will be loaded from JSON config file. It's planned in https://github.com/kubernetes/dashboard/issues/5751.

Reply:
Done.